lib/features/email/presentation/widgets/attachment_item_widget.dart

```diff
@@ -47,7 +47,7 @@ class AttachmentItemWidget extends StatelessWidget {
         return Material(
           color: Colors.transparent,
           child: InkWell(
-            onTap: isLoading ? null : () => viewAttachmentAction?.call(attachment),
+            onTap: isLoading ? null : () => (viewAttachmentAction ?? downloadAttachmentAction)?.call(attachment),
```

Comment:
still not understand, I still see `viewAttachmentAction` and `viewAttachmentAction` in `EmailView` line 346-359